src/current/_includes/v23.1/ui/sessions.md

```diff
@@ -73,6 +73,16 @@ The **Cancel session** button ends the session. The client that holds this sessi
 
   A list of the most recent transaction fingerprint IDs, represented in hexadecimal, executed by this session. The fingerprint ID is a link to the [Transaction Details page]({{ link_prefix }}ui-transactions-page.html#transaction-details-page) for the transaction. When you click the link, in order to fetch and render the transaction details, the global date range is changed to the session's start and end time.
 
+### Closed session cache
+
+Details about closed sessions are stored in memory in a closed session cache. This cache is controlled by the following two [cluster settings]({{ link_prefix }}cluster-settings.html):
+
+- [`sql.closed_session_cache.capacity`] ({{ link_prefix }}cluster-settings.html#setting-sql-closed-session-cache-capacity) is the maximum number of sessions in the cache and defaults to `1000`. Once this limit is reached, older sessions are deleted from the cache so newer sessions can be included.
+- [`sql.closed_session_cache.time_to_live`]({{ link_prefix }}cluster-settings.html#setting-sql-closed-session-cache-time-to-live) is the maximum time to live, in seconds, and defaults to  `3600`. 60 minutes after a session is closed, it is deleted from the cache.
```

Comment:
@maryliag  Is this sentence accurate:

>  60 minutes after a session is closed, it is deleted from the cache.

I ran a test on a dedicated cluster and my closed session details were unavailable after 15 minutes. How would you explain "maximum time to live"?

Reply:
how many other session you had? would it be more than 1000 by any chance? That would be a reason I can think why it got removed before the TTL.
Also confirming this is dedicated, right? Because this is only stored in-memory, so if a node is restarted, the data is lost

Reply:
Correct, on a dedicated cluster with no node restarts, the data is lost within 15 minutes of the session close when there were only 4 closed sessions.

Reply:
In that case, you might wanna open an issue for this, so we can investigate

Reply:
Created issue https://github.com/cockroachdb/cockroach/issues/115301